projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Boost_Cache.php

```diff
@@ -480,6 +488,65 @@ public function invalidate_cache( $action = Filesystem_Utils::REBUILD_ALL ) {
 		return $this->invalidate_cache_for_url( home_url(), $action );
 	}
 
+	/**
+	 * Ignore certain cookies in the cache parameters so cached pages can be served to these visitors.
+	 *
+	 * @param array $parameters - The parameters with the cookies array to filter.
+	 * @return array - The parameters with cookies removed.
+	 */
+	public function ignore_cookies( $parameters ) {
+		static $params = false;
+
+		// Only run this once as it may be called multiple times on uncached pages.
+		if ( $params ) {
+			return $params;
+		}
+
+		// JETPACK_BOOST_IGNORE_COOKIES is a comma-separated list of cookies to ignore.
+		if ( defined( 'JETPACK_BOOST_IGNORE_COOKIES' ) ) {
+			$cookies = explode( ',', constant( 'JETPACK_BOOST_IGNORE_COOKIES' ) );
+			$cookies = array_map( 'trim', $cookies );
+		} else {
+			$cookies = array();
+		}
+
+		/**
+		 * Filters the browser cookies so cached pages can be served to these visitors.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param array $cookies An array of cookie names to remove from the cookie list.
+		 */
+		$cookies = apply_filters(
+			'jetpack_boost_ignore_cookies',
+			array_merge(
+				$cookies,
+				array( 'cf_clearance', 'cf_chl_rc_i', 'cf_chl_rc_ni', 'cf_chl_rc_m', '_cfuvid', '__cfruid', '__cfwaitingroom', 'cf_ob_info', 'cf_use_ob', '__cfseq', '__cf_bm', '__cflb', 'sbsj_' )
```

Comment:
Looks like `sbsj_` is a prefix for a bunch of cookies. It comes from sourcebuster js. While testing this I was getting a cache-miss because WooCommerce uses sourcebuster for order attribution and `sbsj_session` keeps changing. 

We should:
- Add `sbsj_session` to the ignore list.
- Here `sbsj_` is supposed to be a prefix? If so, it's currently not detecting as a prefix which we need to fix.
- Consider if we need to ignore all/more cookies from the `sbsj_` family to the default list.

Reply:
I'm going to change this list to a list regexes. sbsj_ will become sbsj_(.*). The other strings can remain the way they are for simplicity.